tests/baseline/cases/staged-simple-ami-bc.yml

```diff
@@ -34,6 +34,7 @@ items:
       base_product: null
       base_version: null
       date: 2020-05-11
+      marketplace_title: null
```

Comment:
Could you please make at least one of the tests have a non-null value?

Otherwise the tests would pass even if the loading code was broken and always set the value to None.

Reply:
Test added.